src/bsp/src/stm32/include/Decawave.h

```diff
@@ -0,0 +1,163 @@
+#ifndef __DECAWAVE_H__
+#define __DECAWAVE_H__
+
+#include "UWBRxFrame.h"
+#include "deca_device_api.h"
+#include "deca_port.h"
+#include <BaseTask.h>
+#include <FreeRTOS.h>
+#include <functional>
+#include <memory>
+#include <task.h>
+
+enum class DW_LED { LED_0, LED_1, LED_2, LED_3 };
+enum class UWBSpeed { SPEED_110K, SPEED_850K, SPEED_6M8 };
+
+class Decawave {
+  public:
+    explicit Decawave(decaDevice_t spiDevice);
+    Decawave(decaDevice_t spiDevice, uint8_t channel, UWBSpeed speed);
+    ~Decawave() = default;
+
+    /**
+     * @brief Tries communicating by SPI with the DW1000 and configures ISR, GPIO, etc.
+     * @return true if successful, false otherwise.
+     */
+    bool start();
```

Comment:
does it start anything? if it is only a config/init maybe name it that way

Reply:
Changed for init